model_compression_toolkit/common/immutable.py

```diff
@@ -0,0 +1,18 @@
+
+class ImmutableClass(object):
```

Comment:
Comments

Reply:
I'm gonna add comments for everything at the end